### PR TITLE
Added seeNumRecords to Laravel5 module

### PR DESCRIPTION
**Pull request by dmoreno**
This is equivalent to seeNumRecords in DB module

### CODE REVIEW

File: src/Codeception/Module/Laravel5.php [hunk 1]
```diff
@@ -1050,6 +1075,62 @@ protected function findRecord($table, $attributes = [])
...
+    }
+
+    /**
+     * @param string $table
+     *
+     * @return EloquentModel
+     */
+    protected function getQueryBuilderFromTable($table)
+    {
+        return $query = $this->app['db']->table($table);
```
Comment (janhenkgerritsen) [inline src/Codeception/Module/Laravel5.php]: The `$query` variable is unnecessary here

File: src/Codeception/Module/Laravel5.php [hunk 2]
```diff
@@ -1013,20 +1013,45 @@ public function grabRecord($table, $attributes = [])
...
+     * @param string $table
+     * @param array $attributes
+     * @part orm
+     */
+    public function seeNumRecords($expectedNum, $table, $attributes = [])
+    {
+        if (class_exists($table)) {
+            $currentNum = $this->countModels($table, $attributes);
+            if ($currentNum != $expectedNum) {
+                $this->fail("The number of found $table ($currentNum) does not match expected number $expectedNum in with " . json_encode($attributes));
```
Comment (janhenkgerritsen) [inline src/Codeception/Module/Laravel5.php]: Missing text between "in" and "with" in error message.

File: src/Codeception/Module/Laravel5.php [hunk 3]
```diff
@@ -1013,20 +1013,45 @@ public function grabRecord($table, $attributes = [])
...
+            }
+        } else {
+            $currentNum = $this->countRecords($table, $attributes);
+            if ($currentNum != $expectedNum) {
+                $this->fail("The number of found records ($currentNum) do not match expected number $expectedNum in table $table with " . json_encode($attributes));
```
Comment (janhenkgerritsen) [inline src/Codeception/Module/Laravel5.php]: Change "do not match" to "does not match" in error message.

File: src/Codeception/Module/Laravel5.php [hunk 4]
```diff
@@ -1050,6 +1075,62 @@ protected function findRecord($table, $attributes = [])
     }
 
     /**
+     * @param string $modelClass
+     * @param array $attributes
+     * @return array
```
Comment (janhenkgerritsen) [inline src/Codeception/Module/Laravel5.php]: Return type should be integer.

File: src/Codeception/Module/Laravel5.php [hunk 5]
```diff
@@ -1050,6 +1075,62 @@ protected function findRecord($table, $attributes = [])
...
+            $query->where($key, $value);
+        }
+
+        return $query->count();
+    }
+
+    /**
+     * @param string $table
+     * @param array $attributes
+     * @return array
```
Comment (janhenkgerritsen) [inline src/Codeception/Module/Laravel5.php]: Return type should be integer.

**Comment by dmoreno** [comment]
@janhenkgerritsen, thanks for your code review. `grabNumRecords` method is a good idea. I have applied all your proposed changes.

File: src/Codeception/Module/Laravel5.php [hunk 6]
```diff
@@ -1033,17 +1033,38 @@ public function seeNumRecords($expectedNum, $table, $attributes = [])
...
+     * ```
+     *
+     * @param string $table
+     * @param array $attributes
+     * @return integer
+     * @part orm
+     */
+    public function grabNumRecords($table, $attributes = [])
+    {
+        return (class_exists($table))? $this->countModels($table, $attributes) : $this->countRecords($table, $attributes);
```
Comment (janhenkgerritsen) [inline src/Codeception/Module/Laravel5.php]: Can you remove the extra parentheses around the `class_exists` call?

**Comment by dmoreno** [comment]
@janhenkgerritsen, done!

**Comment by janhenkgerritsen** [comment]
Thanks for adding the `grabNumRecords` method and applying the changes. I see you already pushed the change for the parentheses :)

The previous build failed, but this was not related to your changes.

**Comment by janhenkgerritsen** [comment]
I added the changes from this PR to the changelog in #3822. Thanks for your work!

**Pull request merged**